modules/post/windows/gather/enum_computers.rb

```diff
@@ -115,12 +100,28 @@ def list_computers(domain, hosts)
         ]
     )
     hosts.each do |hostname|
-      hostip = resolve_host(hostname)
-      tbl << [domain, hostname, hostip]
+      hostipv4 = gethost(hostname, 2)
+      hostipv6 = gethost(hostname, 23)
```

Comment:
Would it be possible to use named constants instead of magic numbers?

Reply:
Yea, those values were just added for testing initially. Updated to make use of existing constants 👍